evm/bridge-token-factory/scripts/deploy-factory.js

```diff
@@ -0,0 +1,53 @@
+require("dotenv").config();
+const { ethers, upgrades } = require("hardhat");
+
+const BRIDGE_TOKEN_IMPL_ADDRESS = process.env.BRIDGE_TOKEN_IMPL_ADDRESS;
```

Comment:
I don't like this script format. We should use hardhat tasks and config files, as we did for other projects.

Reply:
Done 7ae04b99707283238932465ab7bb228efbb57cb0